docs/advanced/api_examples.yml

```diff
@@ -171,6 +171,43 @@
 
 -
   id:
+    "update dataset with primary"
+
+  doc: |
+    Update a dataset with a new primary address.
+
+  requires:
+    - "create dataset with dataset_id"
+
+  request: |
+    POST /v1/configuration/datasets/886ed03a-5606-453a-94a9-a1cbaf35164c HTTP/1.1
+
+    {"primary": "%(NODE_1)s"}
+
```

Comment:
```
Exception occurred:
  File "/srv/buildslave/flocker-docs/build/flocker/restapi/docs/publicapi.py", line 266, in _formatExample
    lines = (example.request % substitutions).splitlines()
KeyError: 'NODE_1'
```

`NODE_1` is not defined. Relevant definitions at `flocker/restapi/docs/publicapi.py:299`.